changelog.json

```diff
@@ -1,5 +1,10 @@
 {
   "releases": {
+    "1.3.0-beta2": [
+      "[Fixed] Diff gutter elements should be considered button elements when interacting - #5158",
+      "[Fixed] Add option to move repository to trash when removing from app - #2108. Thanks @say25!",
```

Comment:
IMO this should be a `[New]` or `[Added]`.

Reply:
:thumbsup: to `Added`